main/scraping/gainers.py

```diff
@@ -0,0 +1,74 @@
+from bs4 import BeautifulSoup
+import pandas as pd
+
+def daily_gainers():
+    dfs = pd.read_html('https://money.rediff.com/gainers/bse/daily',header=0)
+    for df in dfs[:-1]:
+        df=df
+
+    df['% Change'] = df['% Change'].str.replace(' ', "")
+    df1 = df[['Company', '% Change','Current Price (Rs)']][:10]
+    data = {}
+    col = list(df1)
+    for i in range(10):
+        current = 'Company {}'.format(i+1)
+        data[current] = {}
+        c=0
+        for j in col:
+            if c==0:
+                data[current]['Company Name'] = df[j][i]
+            elif c==1:
+                data[current]['% Change'] = df[j][i]
+            else:
+                data[current]['Current Price (Rs)'] = df[j][i]
+            c+=1
+    
+    return data
+
```

Comment:
@hpb1  functions for determining daily ,weekly  and monthly gainers have been pushed by @Bhard27  , it would be better if we use the same code in scraping/gainers directory 

The only difference seems to be in the way variable c is  updated in each file , please check into this